payments-core/src/main/java/com/stripe/android/view/CardWidgetViewModel.kt

```diff
@@ -71,11 +71,9 @@ internal class CardWidgetViewModel(
         return config?.cardBrandChoice?.eligible ?: false
     }
 
-    class Factory : ViewModelProvider.Factory {
+    class Factory(val context: Context) : ViewModelProvider.Factory {
```

Comment:
Is this necessary to make things work correctly in `Activity`?

Reply:
Yup! `extras.requireApplication()` will crash for base Activities/Dialogs